resources/dicts/names/names.json

```diff
@@ -297,115 +296,114 @@
-    "Mawuli", "Max", "May", "Maya", "Mazu", "McChicken", "McFlurry", "McLovin", "Meadow of Two Worlds", "Meatloaf", "Meatlug", "Medusa", "Megabyte", "Megan", "Meilin", "Meimei", "Melba", "Melody", 
-    "Melon", "Melona", "Memories Lost", "Meow-Meow", "Meowth", "Meowyman", "Mera", "Mercedes", "Mercury", "Mercy", "Merengue", "Merlot", "Merry", "Mew", "Michelle", "Michi", "Mick", 
-    "Midnight Goddess", "Mieke", "Mikumi", "Miles", "Milhouse", "Milkshake", "Milky Way", "Millie", "Milo", "Milque", "Mimas", "Mimi", "Mimikyu", "Minette", "Minha", "Mini", "Minna", 
+    "Mare", "Mareep", "Marie", "Marina", "Mario", "Mariposa", "Mark", "Marlow", "Marny", "Mars", "Marshal", "Marshmallow", "Martini", "Marula", "Mason", "Matador", "Matcha", "Mathias", "Maverick", "Martha",
+    "Mawuli", "Max", "Maxie", "May", "Maya", "Mazu", "McChicken", "McFlurry", "McLovin", "Meadow of Two Worlds", "Meatloaf", "Meatlug", "Medusa", "Megabyte", "Megan", "Meilin", "Meimei", "Melba", "Melody", 
+    "Melon", "Melona", "Memories Lost", "Meow-Meow", "Meowth", "Meowyman", "Mera", "Mercedes", "Mercury", "Mercy", "Merengue", "Merlot", "Merry", "Mew", "Miau", "Michelle", "Michi", "Mick", 
+    "Midnight Goddess", "Mieke", "Mikhail", "Mikumi", "Miles", "Milhouse", "Milkshake", "Milky Way", "Millie", "Milo", "Milque", "Mimas", "Mimi", "Mimikyu", "Minette", "Minha", "Mini", "Minna", 
     "Minnie", "Mint", "Minty", "Mira", "Miranda", "Miriam", "Miso", "Miss Marple", "Missile Launcher", "Misty", "Mitaine", "Mitochondria", "Mitski", "Mitsubishi", "Mittens", 
     "Mitzi", "Mitzy Moo Moo", "Miyun", "Mizan", "Mizar", "Mizu", "Mocha", "Mochi", "Mojito", "Mojo", "Mollie", "Molly Murder Mittens", "Molly", "Monika", "Monster", 
     "Monte", "Monzi", "Moo", "Moon", "Mooncake", "Mop", "Mora", "Morb", "Morbius", "Mordred", "Morel", "Morpeko", "Morphius", "Morrhar", "Morty", "Mosaic", "Mosi", "Mowgli", "Moxie", 
     "Mr. Anderson", "Mr. Kitty Whiskers", "Mr. Kitty", "Mr. Midnight", "Mr. Mistoffolees", "Mr. Mustard", "Mr. Oreo", "Mr. President", "Mr. Princess", "Mr. Sir", "Mr. Whiskers", "Mr. Wigglebottom", 
     "Mucha", "Mudkip", "Mufasa", "Muffet", "Muffy", "Mullido", "Mumbai", "Mungojerrie", "Munkustrap", "Murder", "Mushroom", "Mustard", "Myko", "Nabi", 
-    "Nadia", "Nadir", "Nadja", "Nagi", "Nagito", "Nakala", "Nakeena", "Nala", "Namielle", "Namu", "Nandor", "Naomi", "Nari", "Neapolitan", "Neel", "Nefertiti", "Neil", "Nemo", "Neo", "Nephrite", "Neptune", "Ness", "Nessie", "Neuron", "Neutron", "Nevaeh", 
-    "Neytiri", "Nezuko", "Nibbles", "Nick", "Nidorina", "Nightcat", "Nightmare", "Nikki", "Nile", "Niles", "Nimbus", "Nine", "Ninetales", "Ninja", "Nintendo", "Nintendo DS", "Nintendo Switch", "Nirav", 
+    "Nadia", "Nadir", "Nadja", "Nagi", "Nagito", "Nakala", "Nakeena", "Nala", "Namielle", "Namu", "Nandor", "Naomi", "Nari", "Naughty Wilbur", "Neapolitan", "Neel", "Nefertiti", "Neil", "Nemo", "Neo", "Nephrite", "Neptune", "Ness", "Nessie", "Neuron", "Neutron", "Nevaeh", 
```

Comment:
As much as I love "Naughty Wilbur", it seems strange to add a normal name with an adjective in front of it. Maybe just add "Wilbur"?